pennylane/devices/default_qubit.py

```diff
@@ -591,29 +587,38 @@ def execute(
             if execution_config.gradient_method in {"backprop", None}
             else None
         )
+        prng_keys = [self.get_prng_keys()[0] for _ in range(len(circuits))]
+
         if max_workers is None:
+            simulate_kwargs = {
+                "rng": self._rng,
+                "debugger": self._debugger,
+                "interface": interface,
+                "state_cache": self._state_cache,
+            }
             return tuple(
-                simulate(
+                _simulate_wrapper(
                     c,
-                    rng=self._rng,
-                    prng_key=self.get_prng_keys()[0],
-                    debugger=self._debugger,
-                    interface=interface,
-                    state_cache=self._state_cache,
+                    {
+                        "rng": self._rng,
+                        "debugger": self._debugger,
+                        "interface": interface,
+                        "state_cache": self._state_cache,
+                        "prng_key": _key,
+                    },
                 )
-                for c in circuits
+                for c, _key in zip(circuits, prng_keys)
             )
 
         vanilla_circuits = [convert_to_numpy_parameters(c) for c in circuits]
         seeds = self._rng.integers(2**31 - 1, size=len(vanilla_circuits))
-        _wrap_simulate = partial(simulate, debugger=None, interface=interface)
+        simulate_kwargs = [
+            {"debugger": None, "interface": interface, "rng": _rng, "prng_key": _key}
```

Comment:
```suggestion
            {"rng": _rng, "prng_key": _key}
```
Could potentially just leave these out for multiprocessing.  As debugger will always be the `simulate` default, and `interface` will always be the `"numpy"` default.